source/js/buyers-guide/components/creep-vote/creep-vote.jsx

```diff
@@ -71,7 +71,7 @@ export default class CreepVote extends React.Component {
 
   sendVoteFor(payload) {
     let attribute = payload.attribute;
-    let url = `/api/buyersguide/vote/`;
+    let url = window.location.href;
```

Comment:
ProductPages, when being voted on, will POST to themselves instead of posting to an explicit endpoint.

Reply:
Surely we can post to `.` instead =)

Reply:
Feature flag added. Good catch Pomax!